plugins/task/privacyconsent/src/Extension/PrivacyConsent.php

```diff
@@ -0,0 +1,241 @@
+<?php
+
+/**
+ * @package     Joomla.Plugin
+ * @subpackage  Task.PrivacyConsent
+ *
+ * @copyright   (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Plugin\Task\PrivacyConsent\Extension;
+
+use Joomla\CMS\Application\ApplicationHelper;
+use Joomla\CMS\Factory;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Mail\Exception\MailDisabledException;
+use Joomla\CMS\Mail\MailTemplate;
+use Joomla\CMS\Plugin\CMSPlugin;
+use Joomla\CMS\Router\Route;
+use Joomla\CMS\Uri\Uri;
+use Joomla\CMS\User\UserHelper;
+use Joomla\Component\Scheduler\Administrator\Event\ExecuteTaskEvent;
+use Joomla\Component\Scheduler\Administrator\Task\Status;
+use Joomla\Component\Scheduler\Administrator\Task\Task;
+use Joomla\Component\Scheduler\Administrator\Traits\TaskPluginTrait;
+use Joomla\Database\DatabaseAwareTrait;
+use Joomla\Database\ParameterType;
+use Joomla\Event\SubscriberInterface;
+use PHPMailer\PHPMailer\Exception as phpmailerException;
+// phpcs:disable PSR1.Files.SideEffects
```

Comment:
```suggestion

// phpcs:disable PSR1.Files.SideEffects
```
Seems something went wrong with my previous suggestion. Only 1 of the 2 empty lined needed to be removed. Now adding one back.